README.md

```diff
@@ -28,7 +33,7 @@ Select 1:
 [] Selects all li's which are directly inside a ul of class dropdown (children)
 [] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
 [] Selects all ul's of class dropdown, as well as the children elements that are li's
-[] Selects all ul's of class dropdown, only if their children are exclusively li's
+[x] Selects all ul's of class dropdown, only if their children are exclusively li's
 ```
```

Comment:
Correct answer: Selects all li's which are directly inside a ul of class dropdown (children).

The `>` in a CSS selector means that you are selected the "direct children" of a given element (i.e., one level down).